src/PharoLauncher-CLI-Tests/PhLTemplateInfoCliCommandTest.class.st

```diff
@@ -0,0 +1,60 @@
+Class {
+	#name : #PhLTemplateInfoCliCommandTest,
+	#superclass : #PhLTemplatesCliCommandTest,
+	#category : #'PharoLauncher-CLI-Tests'
+}
+
+{ #category : #tests }
+PhLTemplateInfoCliCommandTest >> testTemplateInfoShouldSucceed [
+	self createOfficialRepositoryInFileSystemWithTemplatesInIt. 
+	
+	self runCommand: #('launcher' 'template' 'info' 'fakeTemplate').
+	
+	self assertSuccess.
+	
+	
+]
+
+{ #category : #tests }
+PhLTemplateInfoCliCommandTest >> testTemplateInfoToManyArguments [
```

Comment:
rename #testTemplateInfoToManyArguments to #testTemplateInfoWithTooManyArgumentsShouldGiveErrorMessage ?